doc/rose-rug-advanced-tutorials-arch.html

```diff
@@ -0,0 +1,368 @@
+source-prefix=$ROSE_DATAC/
+target-prefix=$ARCH_TARGET/
+update-check=mtime+size
+
+[arch:rename/]
+rename-format=%(cycle)s_%(tag)s_%(name)s
+rename-parser=^.*list(?P&lt;tag&gt;.*)$
+source=file_list?
+</pre>
+    </div>
+
+    <div class="slide">
+      <h2 id="description">Description</h2>
+
+      <p>You have now created a suite that:</p>
+
+      <ul>
+         <li><samp>make_files</samp> sets up the files and archving directory
+        for <samp>archive_files</samp> to use.</li>
+
+        <li>contains a <samp>archive_files_rsync</samp> task that "archives", 
+        rsync's files to an ARCHIVING folder in the $ROSE_DATA directory.</li>
+        <li>contains a <samp>archive_files_scp</samp> task that "archives", 
+        scp's renamed files and moves them to an ARCHIVING folder in the 
+        $ROSE_DATA directory.</li>
+                
+      </ul>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Run the suite</h3>
+
+      <p>Save your changes and run the suite using <samp>rose suite-run</samp>
+      .</p>
+
+      <p>View the suite output using <samp>rose suite-log</samp> and inspect
+      the output of the <samp>make_files</samp>,
+       <samp>archive_files_rsync</samp> and <samp>archive_files_scp</samp>
+      tasks.</p>
+    </div>
+
+    <div class="slide">
+      <h3>Results of "archiving"</h3>
+
+      <p>Change to the $ROSE_DATA/ARCHVING directory of the suite i.e.
+      cylc-run/SUITE_ID/share/data/ARCHIVING.</p>
+
+      <p>List the directory by typing:</p>
+      <pre class="shell">
+ls
+</pre>
+      <p>You should see the following returned:</p>
+      <pre class="shell">
+<font color="skyblue">dir</font>  <font color="red">file_zipped.tar</font>  <font color="skyblue">files</font>  <font color="skyblue">rename</font>  solo.file
+</pre>
+      <p>Change to the <em>files</em> directory and list the files, you should
+     see the following returned:</p>
+      <pre class="shell">
+file_list1  file_list3           
+</pre>
+
+      <p>Change back to the ARCHIVING directory and then to the <em>dir</em>
+      directory. List the files, you should see the following returned:</p>
+      <pre class="shell">
+file_list1  file_list2 file_list3 
+</pre>
+      <p>Note this was all the <em>file*</em> files in the 
+      <code>$ROSE_DATA</code> directory.</p>
+
+      <p>Change to the <em>rename</em> directory and list the files, you should
+      see the following returned:
+      <pre class="shell">
+1_1_file_list1 1_2_file_list2 1_3_file_list3 
+</pre>
+      </p>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Other archiving commands, i.e. rcp, moo</h3>
+
+      <p>Most users will have there own system or location that they wish to
+      archive their data to. Here the example shown uses <strong>rsync</strong>
+      and <strong>scp</strong>, other commands that can be used are 
+      <strong>rcp</strong> and for MetOffice the in-house archiving system uses
```

Comment:
Please remove reference on `rcp`. It is a legacy command that is superseded by `scp`.

Please also remove reference on `moo`. Perhaps we can ask users to refer to the documentation for their site specific archiving solution.

The most important message here should be that any commands with a compatible interface can be used.